e2e-tests/cypress/support/appellant-submission-your-details/confirmApplicantNameWasAccepted.js

```diff
@@ -1,4 +1,4 @@
 module.exports = () => {
-  cy.url().should('include', '/appellant-submission/task-list');
+  cy.url().should('include', '/appeal-householder-decision/task-list');
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks